### PR TITLE
Feature: Profile Page

### PR DESCRIPTION
**Pull request by chelsabeth**
Created a way to upload an image in the profile component and did some basic styling. I separated the style sheet for the profile component into the Profile folder that way it did not interfere with the main css. 

Link to Trello Card:  
https://trello.com/c/tqLZJ9ny

### CODE REVIEW

**Review by locbgiang — APPROVED** [review]
Everything looks good.

File: src/components/pages/Profile/Profile.js [hunk 1]
```diff
@@ -1,12 +1,75 @@
...
+  };
+
+  const handleSubmitFile = e => {
+    e.preventDefault();
+    if (!previewSource) return;
+    uploadImage(previewSource);
+  };
+
+  // TODO: connect with backend server to make a post request - 404 error at the moment
+  const uploadImage = async base64EncodedImage => {
```
Comment (vtopham) [inline src/components/pages/Profile/Profile.js:32]: Maybe we can standardize to axios in the future when we go full CRUD :)

File: src/components/pages/Profile/profile.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+form {
```
Comment (vtopham) [inline src/components/pages/Profile/profile.css:1]: Thank you for moving this!

File: src/components/pages/Profile/profile.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  .submit-btn {
+    margin-top: 2%;
+  }
+  
+  .submit-btn {
+    color: white;
+    background-color: #00a6af;
+    border: 1px solid white;
+    border-radius: 4px;
+    height: 2rem;
```
Comment (vtopham) [inline src/components/pages/Profile/profile.css:16]: Love the rem :')

**Review by vtopham — APPROVED** [review]

**Pull request merged**